parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/StatisticsSerializer.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.parquet.hadoop.metadata;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import org.apache.parquet.column.statistics.Statistics;
+
+class StatisticsSerializer extends JsonSerializer<Statistics<?>> {
```

Comment:
Should we add a test?

Reply:
BTW, what is the original behavior?

Reply:
I had 2nd thought on this. Printing the statistics with the corresponding json primitive is not trivial.
As the `sizeStatistics` are not guaranteed to be there, I change to ignore normal stats in the logging